doc/api/esm.md

```diff
@@ -1267,7 +1292,7 @@ The resolver can throw the following errors:
 * _Unsupported Directory Import_: The resolved path corresponds to a directory,
   which is not a supported target for module imports.
 
-### Resolver Algorithm Specification
+### Default resolver Algorithm Specification
```

Comment:
Again, we must keep this as `Resolver Algorithm Specification`. If we want to be more specific then `Node.js Resolver Algorithm`.

Reply:
Can we call this the “Resolution and Loading Algorithm Specification”? And the steps continue into the load phase, which I think is just “if this validation fails, throw this error; if this other validation fails, throw this other error; load the source from disk/network”. Is that about right? Then we don’t need to hide the `load` phase errors in explanatory paragraphs; they can be part of the spec.

Reply:
See above for the problem with these titles.